javascript/house-style.md

```diff
@@ -195,6 +195,46 @@ We don't do this:
 const uniqueIDs = new Set([1, 2, 1, 2]);
 ```
 
+#### JSDoc
+
+Although your JavaScript code should be self-documenting, we highly recommend
+that you document your JavaScript application or library using the
+Standardised [JSDoc](https://jsdoc.app/) format.
```

Comment:
I think this might a bit more clear, as we seem to be suggesting that self-documenting code is the preference while also recommending using JSDoc

```suggestion
Although your JavaScript code should be self-documenting, we highly recommend
that where necessary, you document your JavaScript application or library using the
standardised [JSDoc](https://jsdoc.app/) format.
```